lan-mouse-ipc/src/lib.rs

```diff
@@ -253,6 +256,8 @@ pub enum FrontendRequest {
     RemoveAuthorizedKey(String),
     /// change the hook command
     UpdateEnterHook(u64, Option<String>),
+    /// update the input post-processing settings (invert-scroll, mouse_mod)
+    UpdateInputConfig(InputConfig),
```

Comment:
In foresight to implementing frontends, it will likely be more convenient to split this into two separate requests.
This will also solve the above mentioned dependency issue.